notebooks/MAST/TESS/interm_tess_prf_retrieve/requirements.txt

```diff
@@ -1,4 +1,4 @@
 astropy==2.0.8
-astroquery==0.3.10.dev0
+astroquery==0.3.10
```

Comment:
It is not wise to pin astroquery. Should always use the latest pre-release to pick up fixes for upstream service changes, but that is another story.